presto-hive-common/src/main/java/com/facebook/presto/hive/HiveCommonSessionProperties.java

```diff
@@ -57,6 +58,7 @@ public class HiveCommonSessionProperties
     private static final String ORC_STREAM_BUFFER_SIZE = "orc_stream_buffer_size";
     private static final String ORC_TINY_STRIPE_THRESHOLD = "orc_tiny_stripe_threshold";
     private static final String ORC_ZSTD_JNI_DECOMPRESSION_ENABLED = "orc_zstd_jni_decompression_enabled";
+    private static final String ORC_MAX_COALESCED_DISTANCE_BYTES = "orc_max_coalesced_distance_bytes";
```

Comment:
There is `orc_max_merge_distance` session property for Presto that does exactly the same. I wonder if it would be better to rename the property in Prestissimo / Velox to match the Presto name to avoid unnecessary confusion?

Reply:
@arhimondr 
Great, I didn't know we have that already.
We should definitely use it.

We don't need another one.
It is a different name and of DataSize type, so it will need some translation in Presto C++ layer.

Reply:
@spershin In Velox we would like to keep it as format agnostic, is it possible to do the translation `orc_max_merge_distance -> max-coalesced-distance-bytes` in Presto CPP layer?